### ISSUE TITLE
generate nothing generated 

### ISSUE THREAD

**Issue by ghost** [joelittlejohn/jsonschema2pojo#990]
Problem statement:

I have the following json schema (attached)

When i am generating java classes from json schema using  jsonSchema2pojo cli no output files generated.
No any messages or warnings, no any files, nothing at all.

But, at the same time all online json schema validation tools show me my schema is valid.
 
Whats wrong ?

Sergey

[j-schema.zip](https://github.com/joelittlejohn/jsonschema2pojo/files/3240976/j-schema.zip)

**Comment by ddcruver** [comment]
Definitions are not generated by this library unless they are referenced somewhere. I first went down this route of storing a bunch common pojos in a single schema file that were referenced by other files.  We decided that in the end we would create a schema file for each pojo/enum we wanted to create.

Your schema yes while is valid I assume the type defaults to "null" and therefore it does not create an object for this schema file.  Even if it did though you have not defined any properties so it may create an empty POJO with just the additionalProperties map.
